### PR TITLE
Add whisper extension

### PR DESCRIPTION
**Pull request by huningxin**

### CODE REVIEW

**Comment by huningxin** [comment]
@Honry , PTAL, thanks!

/cc @ibelem

File: js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json
Comment (Honry) [inline js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json:1]: Actually `/resolve/main` sub folders are not necessary. But doesn't matter if it works.

File: js/whisper-demo/utils.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ async function readResponse(response) {
 
 export function log(i) {
     console.log(i);
-    document.getElementById('status').innerText += `\n[${performance.now().toFixed(2)}] ` + i;
+    // document.getElementById('status').innerText += `\n[${performance.now().toFixed(2)}] ` + i;
```
Comment (Honry) [inline js/whisper-demo/utils.js]: This function is designed for showing some status info on the page for users.

I suggest using `console.log()` instead for those info we just want to show in console.

**Review by Honry — COMMENTED** [review]
Thanks Ningxin, LGTM % a nit.

File: js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json
Comment (huningxin) [reply js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json:1]: Do you mean I can just place preprocessor_config.json under ./models/openai/whisper-base/?

Comment (Honry) [reply js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json:1]: Right, all the token config files can just place under `./models/openai/whisper-base/`

Comment (huningxin) [reply js/whisper-demo/models/openai/whisper-base/resolve/main/preprocessor_config.json:1]: I have to keep it for local extension usage. Some sites restrict the access to those json files.

File: js/whisper-demo/utils.js (hunk 1 above)
Comment (huningxin) [reply js/whisper-demo/utils.js]: Fixed in https://github.com/Honry/onnxruntime-inference-examples/pull/1/commits/166eaf16c5b7526172e276ae09ec387bd6cc0aa4

**Review by Honry — APPROVED** [review]

**Pull request merged**